.vscode/setup.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+#Activate Python
+python -i
+
+#Install PIP
+python -m pip --version
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Remove this.
pip comes with python, so if you have `python`, you can assume you have `pip`. But also this command isn't doing anything useful. You are downloading a file via this URL, and saving it as a file called `get-pip.py`. And then doing nothing with it.